worker/deps/libwebrtc/libwebrtc/modules/bitrate_controller/loss_based_bandwidth_estimation.cc

```diff
@@ -173,8 +190,15 @@ void LossBasedBandwidthEstimation::Update(Timestamp at_time,
       !has_decreased_since_last_loss_report_ &&
       (at_time - time_last_decrease_ >=
        last_round_trip_time + config_.decrease_interval);
+  // If packet lost reports are too old, dont increase bitrate.
+  const bool loss_report_valid =
+      at_time - last_loss_packet_report_ < 1.2 * kMaxRtcpFeedbackInterval;
```

Comment:
how the RTCP feedback calculated in mediaosup? If I remember RFC3550 calculate based on assumed bw. If we calculate based on that assumption, and the bw suddenly drop because of congestion collapse than the interval will increase massively which makes this loss_report_valid false. at this moment the interpretation collides with the behaviour of the rest of the code, so right now it does not cause any problem and only beauty suggestion is to add a debug log if loss_report_valid is false, so if we search something related to it in the future we might have a chance to find it easier.

Reply:
This is old attempt of loss based BW estimate. It is disabled by default and is not used. Can be turned on by field trial. So we have:
- Loss v1 BW estimator:
    I have tested it and it was producing bad values.
- Loss v2 BW estimator:
    The one we are trying to use. It does not use RR RTC feedback. I relies only on TWCC RTCP feedback. When enabled, default estimator is ignored.
[- Default Loss estimation:](https://github.com/sarumjanuch/mediasoup/blob/bwe_backport/worker/deps/libwebrtc/libwebrtc/modules/bitrate_controller/send_side_bandwidth_estimation.cc#L522)
    Too simple to be used, as it odes not differentiate between random and burst, and has not very meaningful boundaries.

Reply:
Okay, so I am lost in the code a bit. Does this mean it won't have any effect because another estimator is used after this PR? In this case should this code marked somehow as deprecated?